app/src/main/java/com/example/onsalestore/LaunchActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.example.onsalestore;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import com.example.onsale.R;
+
+public class LaunchActivity extends AppCompatActivity {
+
+    Button btnLogin, btnSign;
```

Comment:
It is a good practice to make these fields private.
```suggestion
    private Button btnLogin, btnSign;
```

Reply:
Done